packages/social-previews/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v1.1.5 (unreleased)
```

Comment:
I generally wouldn't write a version for the unreleased release, because it could as well be a major or a minor release. Not a big deal of course, because we'll edit this line anyway.

Reply:
Personally I think it's better to write a version for an unreleased release, although maybe I should have done like "v1.1.5-alpha" instead.

Here I'm saying my PR is a patch release. If someone else does a minor or major change before the release happens, they should edit the version number to reflect their change.

On the other hand, if I _don't_ put a version number then when someone is doing the release they'll have to look at my change to figure out whether it was patch, minor, or major. They'll have to spend extra time on that, reading the diff to figure out WTF I did so they can determine what kind of change it was.